envs/example/ci-full-rhel/group_vars/all.yml

```diff
@@ -56,6 +63,13 @@ haproxy:
 keystone:
   uwsgi:
     method: port
+  jellyroll: true
+  distro:
+    python_post_deps:
```

Comment:
can we get similar thing added under horizon:
distro:
    python_post_dependencies:
      - name: bb-horizon-customization